includes/in-person-payments/templates/html-in-person-payment-receipt.php

```diff
@@ -183,6 +196,7 @@ function format_price_helper( array $product, string $currency ): string {
 			<p id="application-preferred-name"><?php echo sprintf( '%s: %s', esc_html__( 'Application name', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['application_preferred_name'] ) ) ); ?></p>
 			<p id="dedicated-file-name"><?php echo sprintf( '%s: %s', esc_html__( 'AID', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['dedicated_file_name'] ) ) ); ?></p>
 			<p id="account_type"><?php echo sprintf( '%s: %s', esc_html__( 'Account Type', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['account_type'] ) ) ); ?></p>
+			<p id="credentials"><?php echo esc_html__( 'Powered by WooCommerce', 'woocommerce-payments' ); ?></p>
```

Comment:
I wonder if "credentials" is the right word here. How about simply "powered_by"?